tests/Organization.py

```diff
@@ -569,3 +569,63 @@ def testGetVariable(self):
     def testGetVariables(self):
         variables = self.org.get_variables()
         self.assertEqual(len(list(variables)), 1)
+
+    @mock.patch("github.PublicKey.encrypt")
+    def testCreateActionsSecret(self, encrypt):
+        org = self.g.get_organization("demoorg")
+        # encrypt returns a non-deterministic value, we need to mock it so the replay data matches
+        encrypt.return_value = "M+5Fm/BqTfB90h3nC7F3BoZuu3nXs+/KtpXwxm9gG211tbRo0F5UiN0OIfYT83CKcx9oKES9Va4E96/b"
+        secret = org.create_secret("secret_name", "secret-value", visibility="all")
+        self.assertIsNotNone(secret)
+
+    @mock.patch("github.PublicKey.encrypt")
+    def testCreateDependabotSecret(self, encrypt):
+        org = self.g.get_organization("demoorg")
+        # encrypt returns a non-deterministic value, we need to mock it so the replay data matches
+        encrypt.return_value = "M+5Fm/BqTfB90h3nC7F3BoZuu3nXs+/KtpXwxm9gG211tbRo0F5UiN0OIfYT83CKcx9oKES9Va4E96/b"
+        secret = org.create_secret("secret_name", "secret-value", secret_type="dependabot", visibility="all")
+        self.assertIsNotNone(secret)
+
+    def testOrgGetSecretAssertion(self):
+        org = self.g.get_organization("demoorg")
+        try:
+            org.get_secret(secret_name="splat", secret_type="supersecret")
+        except AssertionError:
+            assert True
```

Comment:
Exceptions are asserted as
```suggestion
        with self.assertRaises(AssertionError) as exc:
            org.get_secret(secret_name="splat", secret_type="supersecret")
        self.assertEqual(str(exc.exception), "...")
```